data/build_training_data.py

```diff
@@ -88,13 +88,13 @@ def get_stories(filename):
 
 output_file_path = "output/text_adventures.txt"
 with open(output_file_path, "w") as output_file:
-    filenames = ["stories/story" + str(i) + ".json" for i in range(0, 93)]
+    # filenames = ["stories/story" + str(i) + ".json" for i in range(0, 93)]
```

Comment:
It probably won't be needed in the future, this commented out code can be removed